frameworks/kafka/src/main/dist/server.properties.mustache

```diff
@@ -92,9 +92,11 @@ socket.request.max.bytes={{KAFKA_SOCKET_REQUEST_MAX_BYTES}}
 authorizer.class.name=kafka.security.auth.SimpleAclAuthorizer
 super.users={{SETUP_HELPER_SUPER_USERS}}
 allow.everyone.if.no.acl.found={{SECURITY_AUTHORIZATION_ALLOW_EVERYONE_IF_NO_ACL_FOUND}}
+{{^SECURITY_KERBEROS_ENABLED}}
```

Comment:
@benclarkwood this is for https://jira.mesosphere.com/browse/INFINITY-3435. Would you prefer that I disable the affected test in this PR and then fix this in a separate one?

Reply:
I went ahead and merged this. More overhead in breaking it out.